discover/portal/articles/140-workflow/00-intro.markdown

```diff
@@ -0,0 +1,136 @@
+# Workflow [](id=workflow)
+
+The ancient Greeks were pretty important, so we decided to name Liferay's
+workflow engine *Kaleo*. In Greek, Kaleo means "called ones," which is
+appropriate for a workflow engine that calls users to participate in a review
+process designed for them. Kaleo workflow allows a user to define any number of
+simple to complex business processes/workflows, deploy them, and manage them
+through a portal interface. The processes have knowledge of users, groups and
+roles. You don't have to write a single line of code to accomplish this: all
+you have to do is create one XML document, or if you're a Liferay Digital
+Experience Platform (DXP) customer, you have access to a nifty visual designer
+for workflows, called the Workflow Designer. 
+
+There are several aspects to workflow that you need to understand: 
+
+- Enabling workflow
+
+- Designing review processes
+
+- Uploading workflow definitions
+
+- Configuring assets to use workflows
+
+- Using workflow with other applications
+
+After all that, you should be familiar with how to use Liferay's Kaleo workflow
+to set up approval process for any kind of content before it is published to
+your portal. 
+
+## What's New with Workflow [](id=whats-new-with-workflow)
+
+There are some noteworthy enhancements to the workflow functionality:
+
+- Duplicate a workflow.
+
+- Preview and restore a prior version.
+
+- Save draft workflows and retain them in an unpublished state.
+
+- All Control Panel workflow functionality is unified under one entry, found at
+    Control Panel &rarr; Configuration &rarr; Workflow.
+
+- Create workflows (using a text editor for Liferay Portal CE users, and the Workflow
+    Designer for Liferay DXP customers) at Control Panel &rarr; Configuration
+    &rarr; Workflow. Just click the Add button.
+
+Create your own workflows from scratch or leverage existing workflows.
+
+## Embedded Workflows [](id=embedded-workflows)
+
+In addition to the Single Approver definition, there are some workflow
+definitions that ship with @proudct@ but are not pre-installed, since they're
+primarily included for test cases. They can be found in the Liferay source code
+in
+`modules/aps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/resources/META-INF/definitions`,
+or in your Liferay installation itself. Open your Liferay installation's
+`osgi/marketplace/Liferay CE Forms and Workflow.lpkg`, and then find and open
```

Comment:
This path has changed to _osgi/portal/com.liferay.portal.workflow.kaleo.runtime.impl.jar_.